src/platform/Tizen/SoftAPManager.h

```diff
@@ -0,0 +1,71 @@
+/*
+ *
+ *    Copyright (c) 2021 Project CHIP Authors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#pragma once
+
+#if CHIP_DEVICE_CONFIG_ENABLE_WIFI
+#include <glib.h>
+#include <softap.h>
+
+namespace chip {
+namespace DeviceLayer {
+namespace Internal {
+
+class SoftAPManager
+{
+public:
+    static void Init(void);
+    static void Deinit(void);
+
+    static CHIP_ERROR IsEnabled(bool * isSoftAPEnabled);
+    static CHIP_ERROR Enable(void);
+    static CHIP_ERROR Disable(void);
+
+    static CHIP_ERROR GetAPState(bool * apState);
+    static CHIP_ERROR SetAPState(bool apState);
```

Comment:
Suggest documenting if there are any particular side-effects or boundary conditions, and what the different errors are.

Reply:
The contents related to softap were removed from this patch.
Thank you for your review.